pkg/acsengine/const.go

```diff
@@ -133,7 +133,7 @@ var KubeConfigs = map[string]map[string]string{
 		"dnsmasq":         "k8s-dns-dnsmasq-amd64:1.13.0",
 		"pause":           "pause-amd64:3.0",
 		"tiller":          DefaultTillerImage,
-		"windowszip":      "v1.6.6intwinnat.zip",
+		"windowszip":      "v1.6.9-3intwinnat.zip",
```

Comment:
I think we should use v1.6.9-1 and not bump up since previous versions are just for experiment and not usable. There is no point to publish previous versions.

Reply:
I'm not sure it really matters as they're internal artifacts and not reference in any user documentation or code. Ultimately what we're declaring here is "use this specific zip artifact when provisioning Windows nodes." Plus, given the long turn around with CDN caching we want to feel able to change this.

Reply:
I agree that this it doesn't matter what filename we use for this release. However, we should fix our process going forward where we test our way through this by referring to non-CDN experimental versions, and then only once we have a verified working artifact do we update to a CDN-consuming destination.

Reply:
The caveat is that there is no good way to tell the difference between bad and good zip artifact from the cluster after it is deployed, and it would be hard for ACI team and us to root cause the issue if not from the zip version. But with CDN long delay, I agree this should not block us and we can improve next time.